src/LazyWAVFiles.jl

```diff
@@ -82,6 +82,15 @@ struct DistributedWAVFile{T,N,L,FS} <: AbstractArray{T,N}
     blockarray::L
     fs::FS
 end
+function LazyWAVFiles.DistributedWAVFile(wavpaths::Vector{String})
+    isempty(wavpaths) && error("Found no wav paths.")
+    files = LazyWAVFile.(wavpaths)
+    fs0 = files[1].fs
+    if !all(x->x.fs==fs0, files)
+        error("WAV files in $folder have different sample rates.")
+    end
```

Comment:
```suggestion
    all(x->x.fs==fs0, files) || error("WAV files in $folder have different sample rates.")
```

Reply:
Just found out a bug. Please remove `$folder` as it's undefined.